tests/test_profiles.py

```diff
@@ -130,19 +142,23 @@ def test_sum_profiles(self):
             modes=np.array([0, 1, 2, 4]), params=np.array([1, 0, -2, 1]), sym="auto"
         )
         sp = pp.to_spline()
-        zp = pp.to_fourierzernike()
+        zp = -pp.to_fourierzernike()
 
-        f = pp + sp - (-zp)
+        f = pp + sp - zp
         x = np.linspace(0, 1, 50)
         f.grid = 50
         np.testing.assert_allclose(f(), 3 * (pp(x)), atol=1e-3)
 
         params = f.params
+        assert params.size == len(sp.params) + len(pp.params) + len(zp.params) + 1
```

Comment:
Where does the extra `+ 1` come from? And why is it not needed in line 180 of the product profile test?

Reply:
It's because in `f = pp + sp - zp`, `-zp` first gets parsed as a scaled profile with scale=-1, then a sum profile. So there's an extra parameter for the -1. In the product profile we don't negate anything so no extra scale parameter.